j2cl-archetypes/simple-project/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>\${groupId}</groupId>
+  <artifactId>\${artifactId}</artifactId>
+  <packaging>war</packaging>
+  <version>\${version}</version>
+
+  <properties>
+    <j2cl.maven.plugin.version>${project.version}</j2cl.maven.plugin.version>
+    <elemental2.version>1.1.0</elemental2.version>
+
+    <jsinterop.base.version>1.0.0</jsinterop.base.version>
+    <j2cl.version>${j2cl.version}</j2cl.version>
+
+    <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
+    <maven.compiler.source>${maven.compiler.target}</maven.compiler.source>
+
+    <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
+    <maven.jetty.plugin.version>9.4.44.v20210927</maven.jetty.plugin.version>
```

Comment:
Likely that we'll want to bring this to the j2cl-archetype parent pom so that updating in one place cleans up all archetypes at once (once we have more than one). Probably okay to be this way for now.